### PR TITLE
Feature/add support for wagtail 2

### PR DESCRIPTION
**Pull request by KaitCrawford**
We need to upgrade to Wagtail 2 but haven't been able to move away from Wagtail Surveys yet.
This adds Wagtail 2 support to Wagtail Surveys

### CODE REVIEW

File: wagtailsurveys/tests/settings.py [hunk 1]
```diff
@@ -53,37 +54,59 @@
...
     'django.contrib.messages.middleware.MessageMiddleware',
     'django.middleware.clickjacking.XFrameOptionsMiddleware',
+]
 
-    'wagtail.wagtailcore.middleware.SiteMiddleware',
-    'wagtail.wagtailredirects.middleware.RedirectMiddleware',
-)
+if WAGTAIL_VERSION < (2, 0):
+    MIDDLEWARE_CLASSES = [
+        'wagtail.wagtailcore.middleware.SiteMiddleware',
```
Comment (m1kola) [inline wagtailsurveys/tests/settings.py:69]: Not sure if it’s critical or not, but previously Wagtail middlewares were at the end of the list.

**Review by m1kola — COMMENTED** [review]
There is one minor comment, but in general looks good to me.

Thank you for submitting this PR :)

File: wagtailsurveys/tests/settings.py (hunk 1 above)
Comment (KaitCrawford) [reply wagtailsurveys/tests/settings.py:69]: Thanks for spotting that. I've corrected it now

**Review by m1kola — APPROVED** [review]

**Comment by m1kola** [comment]
Hi @KaitCrawford. This was included in v0.2.0 which is now [available on pypi](https://pypi.org/project/wagtailsurveys/0.2.0/).

**Comment by KaitCrawford** [comment]
Thank you so much!!

**Pull request merged**